ir/src/constraints/mod.rs

```diff
@@ -264,6 +266,14 @@ impl Constraints {
                     domain,
                 )?;
 
+                // check that random values are not used against main columns
+                // Would it be better to cleate local constant instead of make this one public and import it?
```

Comment:
@Overcastan I'm not sure what you mean here. Can you please elaborate?

Reply:
Currently I'm using `DEFAULT_SEGMENT` constant that was declared in the `graph.rs`. But to do so I had to make it public (I would like to avoid doing that) and import it to this file. I was thinking maybe it would be better to create a new constant inside this file. But, as it seems to me now, it is better to import it than to create the same one.